pkg/operator/csi/csidrivernodeservicecontroller/csi_driver_node_service_controller.go

```diff
@@ -115,13 +119,13 @@ func NewCSIDriverNodeServiceController(
 	).WithSyncDegradedOnError(
 		operatorClient,
 	).ToController(
-		c.name,
-		recorder.WithComponentSuffix("csi-driver-node-service_"+strings.ToLower(name)),
+		c.instanceName, // don't change what is passed here unless you also remove the old FooDegraded condition
```

Comment:
this one actually uses `WithSyncDegradedOnError` so in this case we cannot change the name without a clean up routine.